### ISSUE TITLE
Error when starting app

### ISSUE THREAD

**Issue by jonaspu** [zdavatz/Scan2Post#10]
Appears when starting the app without card reader plugged in

![screen shot 2018-10-26 at 15 15 40](https://user-images.githubusercontent.com/6840940/47568727-2a88ca00-d932-11e8-927a-53c4d8446b42.png)

**Comment by zdavatz** [comment]
1. This message comes with 10.13.5 when no card reader is plugged in.

**Comment by zdavatz** [comment]
This is the JSON
```
{
    administrationData =     {
        cardNumber = 80756015690011011327;
        expiryDate = 1653948000000;
        insurance =         {
            id = 01569;
            name = Arcosana;
        };
    };
    identificationData =     {
        birthDate = 543366000000;
        firstName = Philipp;
        lastName = Wiget;
        sex = 1;
        ssn = 7561243330635;
    };
}
```

**Comment by jonaspu** [comment]
1. Start "Console"
2. Filter by "Scan2Post"
3. Run Scan2Post
Result: 
![screen shot 2018-10-26 at 15 37 54](https://user-images.githubusercontent.com/6840940/47570088-8b65d180-d935-11e8-851a-d7d22d72e360.png)

4. Plug in card & see NSNotification
Result:
![screen shot 2018-10-26 at 15 39 04](https://user-images.githubusercontent.com/6840940/47570119-9fa9ce80-d935-11e8-9691-b731e9d0dc5d.png)

**Comment by zdavatz** [comment]
@jonaspu this is not the JSON we send. Can you send us the JSON file from the Server?

**Comment by zdavatz** [comment]
CSS/Acrosana, SanaCura (TestKarte) -> HTTP 400 Error
Concordia -> noch nicht getestet.

**Comment by zdavatz** [comment]
This is our JSON
```
{
 "insuranceCard" : {
   "identificationData" : {
     "firstName" : "Zeno",
     "lastName" : "Davatz",
     "sex" : 1,
     "ssn" : "7569087687779",
     "birthDate" : 169772400000
   },
   "administrationData" : {
     "expiryDate" : 1798671600000,
     "cardNumber" : "80756014790027369055",
     "insurance" : {
       "id" : "01479",
       "name" : "Mutuel Assurances"
     }
   }
 },
 "username" : "Zeno2",
 "password" : "Zeno22"
}
```

**Comment by zdavatz** [comment]
Testing with 10.13.6. I get https://gist.github.com/zdavatz/d644883d2b431301343e8f8a0d7e9baf

**Comment by zdavatz** [comment]
seems to be solved with 4b076ea308a46f0c0c
